docs/source/integrations/iceberg_versioning.md

```diff
@@ -0,0 +1,229 @@
+# Data versioning with Iceberg
+
+[Apache Iceberg](https://iceberg.apache.org/) is an open table format for analytic datasets. Iceberg tables offer features such as schema evolution, hidden partioning, partition layout evolution, time travel, and version rollback. This guide explains how to use Iceberg tables with Kedro. For this tutorial, we will use [`pyiceberg`](https://py.iceberg.apache.org/) which is a library that allows you to interact with Iceberg tables using Python, without the need of a JVM.
+
+## Prerequisites
+
+You will need to create a `spaceflights-pandas` starter project which contains example pipelines to work with. If you haven't already, you can create a new Kedro project using the following command:
+
+```bash
+kedro new --starter spaceflights-pandas
+```
+
+To interact with Iceberg tables, you will also need the `pyiceberg` package installed. You can install it by adding the following line to your `requirements.txt`:
```

Comment:
I think it may be good to mention the version of `pyiceberg`, since it's a relatively rapid changing library that could break some of the functions soon.

Reply:
Added the current version